specifications/FiniteMonotonic/README.md

```diff
@@ -1,14 +1,8 @@
 These specifications illustrate a technique for taking ordinarily-infinite monotonic systems and turning them finite for the purposes of model-checking.
 Briefly, this is accomplished by limiting the maximum divergence (difference between lowest and highest value) of any monotonic variable across the system, then continually transposing the set of monotonic variables to their lowest value.
-The technique is explained at length in [this](BlogPost.md) blog post.
+Neither this approach nor the older approach have a proof of soundness and completeness.  They have been tested on a few examples and seem to work well in practice.
```

Comment:
Nit: in version-controlled markdown, generally each sentence is put on its own line.